### ISSUE TITLE
Node.js 12 actions are deprecated.

### ISSUE THREAD

**Issue by AlexRuiz7** [tgymnich/fork-sync#117]
Although the action works, the following warning message appears on each run:

```
[sync](https://github.com/AlexRuiz7/wazuh-app/actions/runs/3472047825/jobs/5802348093)
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: tgymnich/fork-sync@v1.6.3
```

[@dependabot](https://github.com/apps/dependabot) has already opened several pull requests automatically to solve this, which are not merged yet.  
- #116

**Comment by zwping** [comment]
look look [this](https://github.com/zwping/fork-sync)

**Comment by zwping** [comment]
~~tgymnich/fork-sync@v1.6.3~~
```
zwping/fork-sync@v1.6.5
```

**Comment by tgymnich** [comment]
should be fixed in 1.7
